### PR TITLE
Add doc with links to all related repos

### PR DESCRIPTION
**Pull request by lucyzhang929**

### CODE REVIEW

File: docs/SqlBindingRelatedRepos.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Azure SQL Bindings for Azure Functions 
+ 
+## Docs 
+[Azure SQL bindings for Functions | Microsoft Docs ](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql?tabs=in-process%2Cextensionv3&pivots=programming-language-csharp)
+  
+## SQL Binding VSCode Extension
```
Comment (chlafreniere) [inline docs/SqlBindingRelatedRepos.md:6]: "Bindings" instead of "Binding"

File: docs/SqlBindingRelatedRepos.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+[Azure/azure-functions-extension-bundles at v4.x-preview (github.com)](https://github.com/Azure/azure-functions-extension-bundles/tree/v4.x-preview)
+ 
+## Python SQL Bindings 
+### Python Library 
+We define SqlRow and SqlRowList in _sql.py and the SqlConverter in sql.py. 
+
+[Azure/azure-functions-python-library: Azure Functions Python SDK (github.com)](https://github.com/Azure/azure-functions-python-library)
+ 
+### Python Worker  
+Need to add end to end tests for Sql Bindings here.
```
Comment (chlafreniere) [inline docs/SqlBindingRelatedRepos.md:32]: SQL instead of Sql

**Review by chlafreniere — APPROVED** [review]

File: docs/SqlBindingRelatedRepos.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Azure SQL Bindings for Azure Functions 
+
```
Comment (Charles-Gagnon) [inline docs/SqlBindingRelatedRepos.md:2]: Would suggest adding a brief one or two sentence description of what this document is

File: docs/SqlBindingRelatedRepos.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Azure SQL Bindings for Azure Functions 
+ 
+## Docs 
+[Azure SQL bindings for Functions | Microsoft Docs ](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql?tabs=in-process%2Cextensionv3&pivots=programming-language-csharp)
```
Comment (Charles-Gagnon) [inline docs/SqlBindingRelatedRepos.md:4]: Remove en-us

And do you need all the parameters? If you're just linking to the page I'd remove them

File: docs/SqlBindingRelatedRepos.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  
+## SQL Bindings VSCode Extension 
+[azuredatastudio/extensions/sql-bindings at main · microsoft/azuredatastudio (github.com)](https://github.com/microsoft/azuredatastudio/tree/main/extensions/sql-bindings)
+[sqltoolsservice/src/Microsoft.SqlTools.ServiceLayer/AzureFunctions at main · microsoft/sqltoolsservice (github.com)](https://github.com/microsoft/sqltoolsservice/tree/main/src/Microsoft.SqlTools.ServiceLayer/AzureFunctions)
```
Comment (Charles-Gagnon) [inline docs/SqlBindingRelatedRepos.md:8]: Would be helpful to have a brief description of what each of these repos is for

File: docs/SqlBindingRelatedRepos.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+ 
+## Templates 
+Currently we have templates for C#, JS, TS, and Python.
```
Comment (Charles-Gagnon) [inline docs/SqlBindingRelatedRepos.md:11]: Probably not necessary to call out the types specifically - it'll be annoying to update later on. Just say something like `Contains templates for the various types of bindings and supported languages, each under a Sql* folder`

File: docs/SqlBindingRelatedRepos.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+[Azure/azure-functions-extension-bundles at v4.x-preview (github.com)](https://github.com/Azure/azure-functions-extension-bundles/tree/v4.x-preview)
+ 
+## Python SQL Bindings 
+### Python Library 
+We define SqlRow and SqlRowList in _sql.py and the SqlConverter in sql.py. 
+
+[Azure/azure-functions-python-library: Azure Functions Python SDK (github.com)](https://github.com/Azure/azure-functions-python-library)
+ 
+### Python Worker  
+Need to add end to end tests for SQL Bindings here.
```
Comment (Charles-Gagnon) [inline docs/SqlBindingRelatedRepos.md:32]: Just `End to end tests for SQL Bindings in the Python Worker` should suffice

**Review by Charles-Gagnon — APPROVED** [review]

File: docs/SqlBindingRelatedRepos.md (hunk 4 above)
Comment (lucyzhang929) [reply docs/SqlBindingRelatedRepos.md:4]: Ah yes, I can use the aka.ms/sqlbindings link instead

**Pull request merged**